packages/backend/server/src/plugins/redis/mutex.ts

```diff
@@ -1,22 +1,13 @@
-import { setTimeout } from 'node:timers/promises';
-
 import { Injectable, Logger } from '@nestjs/common';
-import Redis, { Command } from 'ioredis';
+import { Command } from 'ioredis';
 
-import {
-  BucketService,
-  type GraphqlContext,
-  LockGuard,
-  MUTEX_RETRY,
-  MUTEX_WAIT,
-  MutexService,
-} from '../../fundamentals';
+import { ILocker, Lock } from '../../fundamentals';
+import { SessionRedis } from './instances';
 
 const lockScript = `local key = KEYS[1]
 local clientId = ARGV[1]
-local releaseTime = ARGV[2]
 
-if redis.call("get", key) == clientId or redis.call("set", key, clientId, "NX", "PX", releaseTime) then
+if redis.call("get", key) == clientId or redis.call("set", key, clientId, "NX", "EX", 60) then
```

Comment:
is 60 seconds too long?
the expiration time here is used to clean up the lock when the server logic does not release the lock correctly due to an exception

Reply:
adjust to 500ms in next pr

Reply:
a functionality downtime with 60s is ok imo

Reply:
reasonable, already revert to 60 seconds